app/models/container/purging.rb

```diff
@@ -0,0 +1,5 @@
+class Container < ApplicationRecord
+  module Purging
+    include ArchivedPurgingMixin
+  end
+end
```

Comment:
Is this working out of the box? If so, that is cool.

Thought you'd have to define `scope` or something.